src/main/java/com/lightbend/rp/BasicApp.java

```diff
@@ -67,7 +67,11 @@ public void apply(MavenProject project) {
             labels.add("modules.service-discovery.enabled", "true");
         if(settings.enableAkkaClusterBootstrap) {
             labels.add("modules.akka-cluster-bootstrap.enabled", "true");
+            labels.add("modules.status.enabled", "true");
```

Comment:
Is this all I need to do?

Reply:
That's a part of it, yes. For status and health urls to work, they also need to be enabled in `application.conf`, like this: https://github.com/lightbend/reactive-lib/blob/master/status/src/main/resources/rp-tooling.conf. `sbt-reactive-app` does this by having a merging feature - `application.conf` gets merged with `rp-tooling.conf` provided by libraries, at build time.

Reply:
I wrote a comment about my experience with merging configs here: https://github.com/lightbend/reactive-app-maven-plugin/issues/12